.gitlab-ci.yml

```diff
@@ -93,6 +93,11 @@ test_nearlib:
     script:
     - ./scripts/test_nearlib.sh
 
+lint_nearlib:
+    stage: libs
+    script:
+    - cd nearlib && npm install eslint && npm run lint
```

Comment:
--fix? or that should be done on developer side?

Reply:
there is `npm run fix` on developer side